panda_market/login.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8">
+    <title>Login</title>
+    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="login.css">
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+  
+  
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-ZNGJGTXPJB"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', 'G-ZNGJGTXPJB'); 
+  </script>
+
+  </head>
+
+<body>
+  <div class="login">
+
+    <div class="pandaLogo">
+
+      <img src="img/logo.png" alt="logo" id="logo" onclick="location.href='index.html'">
```

Comment:
onclick 태그로 이동시키는 것보다, a태그를 이용하는 것이 더 적절합니다. (웹 접근성 측면)

```suggestion
    <a href="index.html">
      <img src="img/logo.png" alt="logo" id="logo">
    </a>
```